ansible/roles/operator-pipeline/templates/openshift/tasks/cert-project-check.yml

```diff
@@ -7,22 +7,11 @@ spec:
   params:
     - name: bundle_path
       description: path indicating the location of the certified bundle within the repository
-    - name: pyxis_cert_path
```

Comment:
The step cert-project-check is used by all pipelines to retrieve the cert_project_id from ci.yaml.
Lately, to this script there was added functionality to query Pyxis and retrieve cert project by id.
It is not needed in eg. CI pipeline. Also, it demands ssh-credentials, which should not be needed in the CI pipeline. I had to split this task.